lib/fastlane_core/print_table.rb

```diff
@@ -5,26 +5,47 @@ class << self
       # You can pass an array to `hide_key` if you don't want certain elements to show up (symbols)
       def print_values(config: nil, title: nil, hide_keys: [])
         require 'terminal-table'
-        rows = []
-
-        config.available_options.each do |config_item|
-          value = config[config_item.key]
-          next if value.nil?
-          next if value.to_s == ""
-          next if hide_keys.include?(config_item.key)
 
-          rows << [config_item.key, value]
-        end
+        options = config.nil? ? {} : config.values
+        rows = self.collect_rows(options: options, hide_keys: hide_keys.map(&:to_s), prefix: '')
 
         params = {}
-        params[:rows] = rows
+        params[:rows] = limit_row_size rows
         params[:title] = title if title
+        # params[:style] = {width: 80} # broken with 1.4.5
+        # workaround in the meantime
 
         puts ""
         puts Terminal::Table.new(params)
         puts ""
 
-        return params
+        params
```

Comment:
Out of curiosity, why ? Other methods don't typically use return. Is there a specific style guideline to follow ?